services-directions-models/src/main/java/com/mapbox/api/directions/v5/DirectionsCriteria.java

```diff
@@ -178,6 +178,16 @@
    */
   public static final String EXCLUDE_RESTRICTED = "restricted";
 
+  /**
+   * Exclude all cash only tolls along the returned directions route.
+   */
+  public static final String EXCLUDE_CASH_ONLY_TOLLS = "cash_only_tolls";
+
+  /**
+   * Exclude all unpaved roads along the returned directions route.
```

Comment:
```suggestion
   * Exclude all unpaved roads along the returned directions route.
   * Only available for {@link DirectionsCriteria#PROFILE_DRIVING} and {@link DirectionsCriteria#PROFILE_DRIVING_TRAFFIC}
```

Reply:
It's good idea! But none of other comments mention limitation.
![image](https://user-images.githubusercontent.com/6190346/141984601-751f508c-198a-4998-9923-acd82aa97c7c.png)
I would update all or none 🙂
@LukasPaczos , what do you think about it?

Reply:
Since we don't have an architectural way to enforce this, let's skip the doc in this case and let the server respond with an error.